src/Views/FirmwareView.vala

```diff
@@ -33,7 +33,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         Object (
             icon_name: "application-x-firmware",
             title: _("Firmware"),
-            description: _("Firmware updates provided by device manufacturers can improve performance and fix critical security issues.")
+            description: _("New firmware is released by device manufacturers to improve performance, and fix security issues.")
```

Comment:
"fix security issues" isn't an independent clause, it's part of a list, so it doesn't make sense to add a comma here. I'm unconvinced that this adds a significant amount of clarity that would make it worth it to lose existing translations

Reply:
To me that Oxford comma makes sense, because it isn't always both.
Translations are only fuzzified. Translators don't like vague sentences at all, because it introduces active errors or even vaguer language (without a lot of effort).

Reply:
No the comma is definitely incorrect here. The Oxford comma would come into play if you were listing 3 or more items.

The suggestion here is not necessarily always true though. You're trying to make a definitive statement, but that makes the statement misleading. It's important to update firmware because it could contain these kinds of fixes that are important to users, but it could just as easily contain neither of these things. Saying that firmware updates can contain these kinds of fixes instead of saying they for sure do contain these types of fixes is intentional

Reply:
True, it sometimes introduces a backdoor. I was thinking that if the user is there to do something, the why of why it should be done is the important bit to convey. Will try to think of something that works.
I think explaining "installed on actual hardware" should maybe be here instead of the looking for fw bit.


It isn't incorrect, and it removes ambiguity. That is the point of an Oxford comma. 3 or more is a serialized list, the point is the relation between them, also as it relates to the former or latter.
If for no other reason than you could have added one in "No the comma is definitely incorrect here."